src/scss/constants.scss

```diff
@@ -94,15 +96,24 @@ $spacing--xxl: 30px;
 
 $submit-button-diameter: 44px;
 
-$font-size--xxs: 10px;
+// Relative font sizes
+$font-size--document: 1rem;
+$font-size--parent: 1em;
```

Comment:
These are useful for resetting font sizes or keeping them relative in child elements as well as relative margins and paddings that make sense for cases when text needs to be moved  to a certain distance (`1em`, `2em` etc.)